solidity/dashboard/src/utils/ethereum.utils.js

```diff
@@ -75,3 +75,5 @@ export function displayEthAmount(
     BigNumber.ROUND_DOWN
   )
 }
+
+export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
```

Comment:
Since we've got 0x0... address as const now we could refactor other places where this address is used - in `tbtc-authorization.service.js`:

```
  if (ownerAddress !== "0x0000000000000000000000000000000000000000") {
    // yourAddress is an operator
    operators.set(web3Utils.toChecksumAddress(yourAddress), {
      managedGrantInfo: {},
      isWithdrawableForOperator: true,
    })
  }
```

and also in `OperatorPage.jsx`:

```
const [state, setData] = useFetchData(
    operatorService.fetchDelegatedTokensData,
    {
      stakedBalance: "0",
      ownerAddress: "0x0000000000000000000000000000000000000000",
      beneficiaryAddress: "0x0000000000000000000000000000000000000000",
      authorizerAddress: "0x0000000000000000000000000000000000000000",
    }
  )
```